### PR TITLE
Update 'Quantity' glossary definition to explain acronym SI

### PR DESCRIPTION
**Pull request by wiggitywhitney**
ʕु•̫͡•ʔु ✧  Greetings!

During my studies the 'Quantity' glossary definition confused me because I did not know what the acronym 'SI' meant.  This PR updates the definition to explain what the acronym 'SI' means.

### CODE REVIEW

**Review by sftim — CHANGES_REQUESTED** [review]
I would use a hyperlink on line 18:

> whole-number notation with [SI](https://en.wikipedia.org/wiki/International_System_of_Units) suffixes. Fractional numbers are represented

with the link pointing to https://en.wikipedia.org/wiki/International_System_of_Units

**The rest of this feedback is _optional_ to implement.**

---

You can also link “binary units” (or “binary suffixes”) to https://en.wikipedia.org/wiki/Binary_prefix (and I would do so)

Aside: this page uses the term “unit” where SI uses the term “prefix”.

**Comment by wiggitywhitney** [comment]
Sorry for the messy commit history, I am learning Git.  
ಠ~ಠ

**Comment by sftim** [comment]
@wiggitywhitney if you learn how to squash down to 1 commit: great. If not (or not yet):
/label tide/merge-method-squash

File: content/en/docs/reference/glossary/quantity.md [hunk 1]
```diff
@@ -10,22 +10,23 @@ aka:
...
 - core-object
 ---
- A whole-number representation of small or large numbers using SI suffixes.
+ A whole-number representation of small or large numbers using [SI](https://en.wikipedia.org/wiki/International_System_of_Units) suffixes.
 
 <!--more-->
 
 Quantities are representations of small or large numbers using a compact,
-whole-number notation with SI suffixes.  Fractional numbers are represented
+whole-number notation with [SI](https://en.wikipedia.org/wiki/International_System_of_Units) suffixes.  Fractional numbers are represented
```
Comment (sftim) [inline content/en/docs/reference/glossary/quantity.md:18]: I'm afraid you only need to link for the first occurence of a term, not every occurrence.

**Review by sftim — COMMENTED** [review]
@wiggitywhitney it looks like you have left in some unrelated changes and possibly undone some of the changes you did want to make.

If you're new to Git, the easiest thing to do is to start a brand new pull request where you make the equivalent changes, and then close this one.

/hold
as we shouldn't merge this as it stands

File: content/en/docs/reference/glossary/quantity.md (hunk 1 above)
Comment (wiggitywhitney) [reply content/en/docs/reference/glossary/quantity.md]: (☞ﾟ∀ﾟ)☞  Done!  And I fixed the commit history.

**Comment by wiggitywhitney** [comment]
@sftim I made the changes as requested and fixed the messy commit history.  I think this should be g2g. 
\(^-^)/ Huzzah!

**Comment by divya-mohan0209** [comment]
@wiggitywhitney : Please could you squash & merge the commits into one (keeping only the latest changes, of course)?

**Comment by wiggitywhitney** [comment]
@divya-mohan0209  It is done! \(^-^)/

**Review by sftim — COMMENTED** [review]
Thanks

Nice improvement

/lgtm
/approve

**Pull request merged**